torch/_inductor/ir.py

```diff
@@ -5321,6 +5317,22 @@ def codegen(self, wrapper):
 
         # Call to kernel
         self.codegen_comment(wrapper)
+
+        """
+        Filter out None args.
+
+        see _precompile_config(...) and https://github.com/pytorch/pytorch/issues/115344
```

Comment:
If we want to refer to `_precompile_config` here, let's mention it's in the `triton_heuristics.py`?